lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt

```diff
@@ -104,7 +104,7 @@ internal open class AndroidExecutor(
         } else {
             // TODO make measurement only once
             val rawData = "wm size".adbShell()
-            val rawSize = rawData.split(" ")[1].split("x")
+            val rawSize = rawData.replace("Physical size: ", "").split("x")
```

Comment:
I think we should add description what is "Physical size" and why we replase it

Reply:
standard output for `wm size` command